src/lib/server/database/index.test.ts

```diff
@@ -62,6 +62,12 @@ it('should complete a full user journey', async () => {
     expect(await db.addDeptAgent(0, uid)).toStrictEqual(db.AddDeptAgentResult.NoDept);
     expect(await db.addDeptAgent(did, uid)).toStrictEqual(db.AddDeptAgentResult.Success);
 
+    expect(await db.removeDeptAgent(0, nonExistentUser)).toStrictEqual(false);
+    expect(await db.removeDeptAgent(did, nonExistentUser)).toStrictEqual(false);
+    expect(await db.removeDeptAgent(0, uid)).toStrictEqual(false);
+    expect(await db.removeDeptAgent(did, uid)).toStrictEqual(true);
+    expect(await db.addDeptAgent(did, uid)).toStrictEqual(db.AddDeptAgentResult.Success);
```

Comment:
Instead of returning Booleans, it would be better if we return `dept_agents.head` (i.e., `BOOLEAN`) or some `const enum`. Essentially, if successfully removed, let's return whether the removed agent was a department head or not.

Otherwise, if we cannot find the department, return `RemoveDeptAgent.NoDept`. If we cannot find the user, return `RemoveDeptAgent.NoUser`.